Status: sent-upstream

---
From 56d2a4097f4b639a16902390d9bdd8d1350f948c Mon Sep 17 00:00:00 2001
From: oxalica <oxalicc@pm.me>
Date: Sun, 2 Oct 2022 08:09:57 +0800
Subject: [PATCH] Fix tests for neovim 0.8.0

Since neovim/neovim#8519, `NVIM_LISTEN_ADDRESS` is only directly used
when it contains colons or (back-)slashes. Otherwise it is concatanated
with a tempdir prefix and suffixes to form the final path.
---
 tests/test_nvr.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_nvr.py b/tests/test_nvr.py
index 97e08ab..ea3e7f4 100755
--- a/tests/test_nvr.py
+++ b/tests/test_nvr.py
@@ -18,7 +18,7 @@ def run_nvr(cmdlines, env):
         nvr.main(cmdline, env)

 def setup_env():
-    env = {'NVIM_LISTEN_ADDRESS': 'pytest_socket_{}'.format(uuid.uuid4())}
+    env = {'NVIM_LISTEN_ADDRESS': './pytest_socket_{}'.format(uuid.uuid4())}
     env.update(os.environ)
     return env

--
2.39.0

